addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -332,24 +479,73 @@ public void initialize() {
     @Override
     public void onTrippedForceAlarm(ZoneMinderTriggerEvent event) {
         try {
-            logger.info("{}: Received forceAlarm for monitor {}", getLogIdentifier(), event.getMonitorId());
-            Channel channel = this.getThing().getChannel(ZoneMinderConstants.CHANNEL_MONITOR_DETAILED_STATUS);
-            Channel chEventCause = this.getThing().getChannel(ZoneMinderConstants.CHANNEL_MONITOR_EVENT_CAUSE);
+            logger.debug("{}: context='onTrippedForceAlarm' Received forceAlarm for monitor {}", getLogIdentifier(),
+                    event.getMonitorId());
+
+            if (!isThingOnline()) {
+                logger.warn("{}: context='onTrippedForceAlarm' Skipping event '{}', because Thing is 'OFFLINE'",
+                        getLogIdentifier(), event.toString());
+                return;
+            }
+
+            IZoneMinderEventData eventData = null;
 
             // Set Current Event to actual event
             if (event.getState()) {
-                startPriorityRefresh();
+                IZoneMinderConnectionHandler connection = null;
+                try {
+                    connection = aquireSession();
+                    IZoneMinderMonitor monitorProxy = ZoneMinderFactory.getMonitorProxy(connection, getZoneMinderId());
+                    eventData = monitorProxy.getEventById(event.getEventId());
+
+                    logger.debug("{}: URL='{}' ResponseCode='{}' ResponseMessage='{}'", getLogIdentifier(),
+                            eventData.getHttpRequestUrl(), eventData.getHttpStatus(),
+                            eventData.getHttpResponseMessage());
+
+                } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.